packages/backend-plugin-api/api-report.md

```diff
@@ -24,6 +25,64 @@ import { Readable } from 'stream';
 import { Request as Request_2 } from 'express';
 import { Response as Response_2 } from 'express';
 
+// @public
+export type AuditEvent = [
+  message: string,
+  meta: {
```

Comment:
is there some particular distinction that guides what is meta and what is not meta? Does this format follow any standard (like some ISO definition or common cloud format or so)?

Reply:
Yes - https://github.com/backstage/backstage/blob/master/beps/0011-event-auditor/README.md#goals